app/assets/stylesheets/application.css

```diff
@@ -194,6 +194,15 @@
   margin-top: 100px;
 }
 
+/* .field {
+  width: 40vw;
+  padding: 10px;
+  font-size: 16px;
+  margin-bottom: 15px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+} */
+
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused lines of code in order to have a clean code base.